sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamAggregationRel.java

```diff
@@ -82,9 +82,44 @@ public BeamAggregationRel(
     this.windowFieldIndex = windowFieldIndex;
   }
 
+  private NodeStats computeWindowingCostEffect(NodeStats inputStat) {
```

Comment:
nit: it's usually a convention to organize code from up to down for readability, so I suggest moving `computeWindowingCostEffect` below `estimateNodeStats`

Reply:
Same in other rels